### DISCUSSION TITLE
Suggestion: Alternative inclusive colour schemes

### DISCUSSION THREAD

**Discussion by jcwomack** [very-good-science/data-hazards#165] [Feedback]
Hello,

Thanks for creating the Data Hazards project. I think this a great initiative.

I would like to suggest that the Data Hazards labels themselves are made available in multiple colour schemes, including colour schemes that are inclusive for colour blind individuals.

I am colour blind myself ([protanomaly](https://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/)) and find the [current red/green colour scheme used in the labels](https://datahazards.com/contents/data-hazards.html) makes them slightly uncomfortable to view and interpret.

It would be nice if the labels were made available in some different colour schemes that are inclusive for individuals with colour blindness or other visual differences. I am not sure what the most inclusive colour schemes would be, though as a starting point I think monochrome and high-contrast versions would probably be good.

Thanks!

**Comment by ninadicara** [comment]
Closing this as a Discussion and have transferred the suggestion to an Issue as we're streamlining the collection of feedback a bit!
